src/renderer/init.js

```diff
@@ -60,17 +56,14 @@ async function init() {
     connect,
   });
 
-  db.init(userDataDirectory);
-  db.registerTransform("app", "accounts", {
-    get: decodeAccountsModel,
-    set: encodeAccountsModel,
-  });
   const store = createStore({ dbMiddleware });
 
-  const settings = await db.getKey("app", "settings");
-  store.dispatch(fetchSettings(settings));
+  const query = querystring.parse(global.location.search);
+  const initialSettings = JSON.parse(query["?settings"]);
```

Comment:
I'm not fan of using location to provide the initial settings. We need a proper way to send the initial data.

Reply:
I could replace it with a blocking invoke during renderer init, wdyt ?

Reply:
i'm ok with this idea